src/_pytest/reports.py

```diff
@@ -148,6 +148,12 @@ def head_line(self):
             fspath, lineno, domain = self.location
             return domain
 
+    def get_verbose_word(self, config):
```

Comment:
We probably better make this private

Reply:
Makes sense for now, but the idea with my commnet on #5047 is to make this a public API in the end.

But we can interate towards this internally, which makes sesnse.

So I'll change this to `_get_verbose_word` for now then?

Reply:
Sounds good